targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/sdk/ublox-odin-w2-drivers/OdinWiFiInterface.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#ifdef MBED_CONF_RTOS_PRESENT
```

Comment:
I don't think it should be gated by the RTOS, but by either LWIP or some other networking flag.

Reply:
Please review [this](https://github.com/ARMmbed/mbed-os/compare/9f812408902c74902b9b264a6b0d5bda338b9f87..d7b54c07e1dfe2c779e45810568bd207588f5c7f).